site/content/installation/installing-ngf/manifests-openshift.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Installation with Kubernetes manifests on Openshift"
+description: "Learn how to install, upgrade, and uninstall NGINX Gateway Fabric using Kubernetes manifests on Openshift."
+weight: 200
+toc: true
+docs: "DOCS-0000"
+---
+
+{{<custom-styles>}}
+
+## Prerequisites
+
+To complete this guide, you'll need to install:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/), a command-line interface for managing Kubernetes clusters.
+
+## Deploy NGINX Gateway Fabric
+
+Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps. With manifests, you can configure your deployment exactly how you want. Manifests also make it easy to replicate deployments across environments or clusters, ensuring consistency.
+
+- If you’d like to use NGINX Plus:
+  1. To pull from the F5 Container registry, configure a docker registry secret using your JWT token from the MyF5 portal by following the instructions from [here](https://docs.nginx.com/nginx-gateway-fabric/installation/ngf-images/jwt-token-docker-secret). Make sure to specify the secret in the `imagePullSecrets` field of the `nginx-gateway` ServiceAccount.
+  1. Alternatively, pull an NGINX Gateway Fabric image with NGINX Plus and push it to your private registry by following the instructions from [here]({{<relref "installation/ngf-images/pulling-ngf-image.md">}}).
+  1. Update the nginx container's `image` field of the `nginx-gateway` Deployment accordingly.
+
+### 1. Install the Gateway API resources
+
+{{<include "installation/install-gateway-api-resources.md" >}}
+
+### 2. Deploy the NGINX Gateway Fabric CRDs
+
+#### Stable release
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.2.0/deploy/crds.yaml
+```
+
+#### Edge version
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/main/deploy/crds.yaml
+```
+
+### 3. Deploy NGINX Gateway Fabric
+
+{{<note>}}By default, NGINX Gateway Fabric is installed in the **nginx-gateway** namespace. You can deploy in another namespace by modifying the manifest files.{{</note>}}
+
+#### Stable release
+
+##### For NGINX
+
+```shell
+kubectl apply -f https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.2.0/openshift-nginx-gateway.yaml
```

Comment:
Would require publishing something named around this in our releases, is that alright?

Reply:
Yes, that would be in .goreleaser.yml

Reply:
Decided offline to not publish these manifests on release and only publish an SCC file.